src/screens/Library/index.tsx

```diff
@@ -184,9 +281,9 @@ export default function Library(): JSX.Element {
 
         <h3 className="libraryHeader">{titleWithIcons()}</h3>
 
-        {refreshing && <UpdateComponent inline />}
+        {refreshing && !refreshingInTheBackground && <UpdateComponent inline />}
```

Comment:
I added this `refreshingInTheBackground` to be able to differenciate when the library was being refreshed in the background and in the foreground